src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordToString.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// The record includes a synthesized override of object.ToString().
+    /// For `record R(int I) { public int J; }` it prints `R { I = ..., J = ... }`.
+    /// 
```

Comment:
nit: there's trailing whitespace on many lines in this summary block.

Reply:
If this matters to you, consider installing an extension such as Trailing Spaces for VS Code in order to spot it and clean it up more easily.